src/auth/user-import-builder.ts

```diff
@@ -47,6 +47,20 @@ interface SecondFactor {
   factorId: string;
 }
 
+interface UserProviderRequest {
+  uid: string;
+  displayName?: string;
+  email?: string;
+  phoneNumber?: string;
+  photoURL?: string;
+  providerId: string;
+}
+
+interface UserMetadataRequest {
+  lastSignInTime?: string;
+  creationTime?: string;
+  lastRefreshTime?: string|null;
```

Comment:
I am not sure `lastRefreshTime` is supported for import operations. Please remove this one for now.

Reply:
Thanks -- are you referring to the specific field or should I remove the entire interface?

Reply:
Just this specific field.

Reply:
@bojeil-google we have the following in our typings for `UserMetadataRequest`:

```
lastRefreshTime?: string|null;
```

Should we remove that too?